bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -52,7 +52,7 @@
  * The {@link EvccHandler} is responsible for handling commands, which are
  * sent to one of the channels.
  *
- * @author Florian Hotze - Initial contribution
+ * @author Florian Hotze - Initial contribution; Avoid data type issues by using float instead of int
```

Comment:
I think this change is not needed as you are already mentioned as original contributor. Only reason for change would be another contributor making a substantial contribution: https://www.openhab.org/docs/developer/guidelines.html#c-documentation

@openhab/add-ons-maintainers - please correct me if I'm wrong here.

Reply:
I am not sure, but at JS Scripting, we also did some smaller changes without adding a new author comment. I'll remove that.

The contribution guidelines also say:

> Data-transfer-objects (DTOs map from JSON/XML to Java classes) do not require JavaDoc.